specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/custom-words.txt

```diff
@@ -0,0 +1 @@
+autologger
```

Comment:
The custom words need to be added into the custom-words.txt at the root directory of this repo instead of the file here.